source/fundamentals/logging.txt

```diff
@@ -0,0 +1,298 @@
+=======
+Logging
+=======
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. _logging:
+
+Overview
+--------
+
+.. What is on this page?
+
+In this guide, we show you how to set up and configure a logger in the
+MongoDB Java driver.
+
+.. What do any new terms mean?
+
+.. Not applicable. We are treating definition of logger as understood.
+
+.. What can you expect to see on this page?
+
+You will learn how to:
+
+- Set up a logger using the Simple Logging Facade For Java (SLF4J)
+- Configure the log level of your logger
+
+.. Who should read this? Where should I go if this isn't the page I was looking for?
+
+.. Not Applicable. No natural redirect location.
+
+Set Up a Logger
+---------------
+
+This section gives background on the dependencies necessary to set up a
+logger and provides an example logger setup. 
+
+Background
+~~~~~~~~~~
+
+The MongoDB Java driver uses the Simple Logging Facade For Java (SLF4J).
+SLF4J allows you to specify your logging framework of choice at deployment time.
+For more information on SLF4J,
+`see the SLF4J documentation <http://www.slf4j.org/>`__.
+
+Setting up a logger is optional. When you start your application the MongoDB
+Java driver looks for the ``slf4j-api`` package in your classpath. If the driver
+can't find the ``slf4j-api`` package, the driver logs the following warning with
+``java.util.logging`` and disables all further logging:
+
+.. code-block:: none
+
+   WARNING: SLF4J not found on the classpath.  Logging is disabled for the 'org.mongodb.driver' component
+
+To set up a logger you must add the ``slf4j-api`` package, a logging framework, and a
+**binding** to your project's dependency list. A binding is a piece of code that
+connects the ``slf4j-api`` package with a logging framework. The example below shows
+how to bind the ``slf4j-api`` package to the two most popular logging frameworks, Log4j2 and
+Logback.
+
+Example
+~~~~~~~
+
+This example shows how to set up your logger using Maven. Click the
+tab corresponding to the logging framework you would like to use in your project.
+
+.. tabs::
+
+   .. tab:: Logback
+      :tabid: Logback-binding
+
+      Add the following dependencies to your ``pom.xml`` file.
+
+      .. code-block:: xml
+
+         <!-- SLF4J API -->
+         <dependency>
+           <groupId>org.slf4j</groupId>
+           <artifactId>slf4j-api</artifactId>
+           <version>1.7.30</version>
+         </dependency>
+         <!-- This package contains an SLF4J-Logback binding -->
+         <dependency>
+           <groupId>ch.qos.logback</groupId>
+           <artifactId>logback-classic</artifactId>
+           <version>1.2.3</version>
+         </dependency>
+         <!-- Logback core functionality -->
```

Comment:
Nit: As this is already a dependency of `logback-classic` its probably not required to be explicitly added.

Reply:
Thank you for the note. Good catch. I will remove the `logback-classic` dependency. 

It seems like [`logback-classic`](https://mvnrepository.com/artifact/ch.qos.logback/logback-classic/1.2.3) also has `slf4j-api` as a dependency. Would you like the `slf4j-api` package removed from the dependency list? I just tested this in my local app and it seems to be working well.